docker-compose.server.yml

```diff
@@ -0,0 +1,68 @@
+version: "3.4"
+
+services:
+    redis:
+        image: redis:latest
+        hostname: redis
+        networks:
+            - main
+        ports:
+            - "6379:6379"
+        restart: always
+
+    app:
+        image: cornellappdev/app:${IMAGE_TAG}
+        hostname: app
+        env_file: .env
+        command: sh -c "python manage.py makemigrations && python manage.py migrate && gunicorn pear.wsgi -b 0.0.0.0:8000"
+        depends_on:
+            - redis
+        deploy:
+            replicas: 2
+            restart_policy:
+                condition: on-failure
+        expose:
+            - "8000"
+        networks:
+            - main
+        restart: always
+        volumes:
+            - static:/static
+
+    celery_worker:
+        image: cornellappdev/app:${IMAGE_TAG}
+        hostname: celery_worker
+        env_file: .env
+        command: sh -c "./wait-for-it.sh redis:6379 && ./wait-for-it.sh app:8000 -- celery -A pear worker -l info"
+        depends_on:
+            - app
+            - redis
+        deploy:
+            replicas: 3
+            restart_policy:
+                condition: on-failure
+        networks:
+            - main
+        restart: always
+
+    nginx:
+        image: cornellappdev/nginx
```

Comment:
this doesn't actually work so i've been using `alannazhou/nginx:faster` but we'll figure something out later